kedro/template/{{ cookiecutter.repo_name }}/docs/source/conf.py

```diff
@@ -47,7 +47,7 @@
 #
 import re
 
-from {{ cookiecutter.python_package }} import __version__ as release
```

Comment:
Could you make sure this PR contains only changes related to the addition of `JSONBlobDataSet`?